src/Database/Driver.php

```diff
@@ -297,22 +297,48 @@ public function isConnected()
     }
 
     /**
+     * Sets whether or not this driver should automatically quote identifiers
+     * in queries.
+     *
+     * @param bool $enable Whether to enable auto quoting
+     * @return $this
+     */
+    public function enableAutoQuoting($enable)
+    {
+        $this->_autoQuoting = (bool)$enable;
+
+        return $this;
+    }
+
+    /**
+     * Returns whether or not this driver should automatically quote identifiers
+     * in queries
+     *
+     * @return bool
+     */
+    public function isEnabledAutoQuoting()
```

Comment:
It is, however, consistent <3. I'd either keep it like it is, or for the getters move enabled to the tail like you suggest. `getAutoQuotingEnabled` would also work (for consistency sake).